platform/features/table/src/controllers/MCTTableController.js

```diff
@@ -165,17 +165,8 @@ define(
 
             //No need to scroll
             if (this.$scope.displayRows.length < this.maxDisplayRows) {
-                //Check whether need to resynchronize visible with display
-                // rows (if data added)
-                if (this.$scope.visibleRows.length !==
-                    this.$scope.displayRows.length){
```

Comment:
This was to address a specific issue, but I don't remember what. The code that calls this function has changed quite a lot since it was written, so it's possible that the issue was resolved in some other way. 

The change looks ok to me, I can't think of a regression that this would introduce, so if you've tested it, then I'm happy to merge.